README.md

```diff
@@ -1,10 +1,13 @@
 # Systematic Review Datasets
 
-This repository provides an overview of labeled datasets used for Systematic Reviews. The
-datasets are available under an open licence and can be used for text mining and machine
+This repository provides an overview of pre-labeled datasets used for Systematic Reviews. The
```

Comment:
What's the difference between "pre-labeled" and "labeled"?

Reply:
Good question, not sure! In the blogpost I portrayed it as a dataset that has been labeled previously, but actually pre-labeled is the same as labeled then?